src/prefect/cli/work_pool.py

```diff
@@ -83,47 +94,78 @@ async def create(
     """
     if not name.lower().strip("'\" "):
         exit_with_error("Work pool name cannot be empty.")
```

Comment:
There was an example, but I enhanced it and added another one.

Reply:
Oh I meant an example in the error message!

Reply:
Ah, this error only fires if you provide empty quotes (e.g. `prefect work-pool create ''`). Otherwise you get the standard `typer` error: 
```
Usage: prefect work-pool create [OPTIONS] NAME
Try 'prefect work-pool create --help' for help.
╭─ Error ─────────────────────────────────────────────────────────╮
│ Missing argument 'NAME'.                                        │
╰─────────────────────────────────────────────────────────────────╯
```
I think the error message as it is is clear enough.